### PR TITLE
Integration check

### PR DESCRIPTION
**Pull request by trapacska**

### CODE REVIEW

File: _tests/integration/integration_test.go [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+	tag      string
+	abi      string
+}
+
+func TestIsLegacyAVDManager(t *testing.T) {
+	_, err := avdmanager.IsLegacyAVDManager(os.Getenv("ANDROID_HOME"))
+	require.NoError(t, err)
+}
+
+func TestIntegrity(t *testing.T) {
```
Comment (godrei) [inline _tests/integration/integration_test.go]: `TestIntegrity` -> `TestCreateAndStartEmulator`

File: _tests/integration/integration_test.go [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+		startEmulator(t)
+	}
+}
+
+func getEmulatorConfigList() []emulator {
+	return []emulator{
+		emulator{platform: "android-24", tag: "google_apis", abi: "armeabi-v7a"},
+		emulator{platform: "android-25", tag: "android-wear", abi: "armeabi-v7a"},
+		emulator{platform: "android-23", tag: "android-tv", abi: "armeabi-v7a"},
+		emulator{platform: "android-24", tag: "default", abi: "armeabi-v7a"},
```
Comment (godrei) [inline _tests/integration/integration_test.go]: android-24 -> android-19?

File: _tests/integration/integration_test.go [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+
+func startEmulator(t *testing.T) {
+	log.Printf("Validate AVD image")
+
+	avdImages, err := listAVDImages()
+	require.NoError(t, err)
+
+	if !sliceutil.IsStringInSlice(testEmulatorName, avdImages) {
+
+		if len(avdImages) > 0 {
```
Comment (godrei) [inline _tests/integration/integration_test.go]: why do we print the available image? it is just a test...

File: _tests/integration/integration_test.go [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+	// ---
+
+	androidSdk, err := sdk.New(os.Getenv("ANDROID_HOME"))
+	require.NoError(t, err)
+
+	adb, err := adbmanager.New(androidSdk)
+	require.NoError(t, err)
+
+	//
+	// Print running devices Info
```
Comment (godrei) [inline _tests/integration/integration_test.go]: i think this we do not need this section in tests

**Comment by viktorbenei** [comment]
Aborted the `/push` ones + configured protected branch for `master` - no need for both `/pr` and `/push` for every "internal" PR

**Pull request merged**